crmsh/bootstrap.py

```diff
@@ -752,6 +752,7 @@ def init_ssh():
                 not confirm("/root/.ssh/id_rsa already exists - overwrite?"):
             return
         rmfile("/root/.ssh/id_rsa")
+        rmfile("/root/.ssh/authorized_keys")
```

Comment:
This step seems destructive when confirm to overwrite.
But if not rm authorized_keys, the file might still include peer node's public key, then when peer node joining, peer node detect that no need password to login to init node, so peer node will not use init node's new generated keys

Reply:
I don't like this. You are basing what to do in order to make your solution work. If this file creates issues in your new approach, I don't think the solution is to remove this file, we need to think about other solution first.
We already started wrong `crmsh` using `/root/.ssh` keys, let's avoid interfiering more in the machines environment.
I don't that this kind of actions will bring issues in the future.

Reply:
removing a 3rd party file should not be the case. I agree on this